tests/load/test_read_interfaces.py

```diff
@@ -748,7 +748,10 @@ def test_ibis_dataset_access(populated_pipeline: Pipeline) -> None:
     from dlt.helpers.ibis import SUPPORTED_DESTINATIONS
 
     # check correct error if not supported
-    if populated_pipeline.destination.destination_type not in SUPPORTED_DESTINATIONS:
+    if not any(
```

Comment:
`SUPPORTED_DESTINATIONS` are misleading, now they contain configurations. maybe you should rename or get rid of this